pkg/plugin/plugin.go

```diff
@@ -26,8 +27,11 @@ var _ api.Plugin = &plugin{}
 func NewPlugin(entry *logrus.Entry, pluginConfig api.PluginConfig) api.Plugin {
 	log.New(entry)
 	return &plugin{
-		entry:  entry,
-		config: pluginConfig,
+		entry:           entry,
+		config:          pluginConfig,
+		configUpgrader:  config.NewSimpleUpgrader(entry),
```

Comment:
I don't know why we have config.Upgrader - it never did anything and I think we now know we won't need it. #556.